### DISCUSSION TITLE
link_preview_options in sending messages is not working

### DISCUSSION THREAD

**Discussion by bhavesh-ram** [telegraf/telegraf#1959] [Q&A] [answered]
`const token =''
const { Telegraf } = require('telegraf')
const bot = new Telegraf(token)
const userID = XXXX

bot.start((ctx) => ctx.reply('Welcome!'))
bot.command('hello', (ctx) => ctx.telegram.sendMessage(userID, 'Hello!',{
    link_preview_options: {
        is_disabled: false,
        url: 'https://blog.hubspot.com/hs-fs/hubfs/image8-2.jpg?width=600&name=image8-2.jpg'
    }
}).then((res) => {
    console.log(res)
}).catch((err) => {
    console.log(err)
}))
bot.launch()
`

Here I have used this code but in response I'm not getting link_preview_options

**Comment by MKRhere** [comment, marked as answer]
This has been fixed in v4.16.3!

**Reply by bhavesh-ram** [reply]
Thank you for the update! It's working now..😇
